From: Takashi Iwai <tiwai@suse.de>
Subject: kABI fix for hda_codec.relaxed_resume flag
Patch-mainline: Never, kABI workaround
References: bsc#1111666

The recent fix
  ALSA-hda-Don-t-resume-forcibly-i915-HDMI-DP-codec.patch
introduced a new bit flag to hda_codec object, and we need to wrap it
with the usual ifdef for kABI compatibility.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 include/sound/hda_codec.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/sound/hda_codec.h
+++ b/include/sound/hda_codec.h
@@ -262,7 +262,9 @@ struct hda_codec {
 	unsigned int auto_runtime_pm:1; /* enable automatic codec runtime pm */
 	unsigned int force_pin_prefix:1; /* Add location prefix */
 	unsigned int link_down_at_suspend:1; /* link down at runtime suspend */
+#ifndef __GENKSYMS__
 	unsigned int relaxed_resume:1;	/* don't resume forcibly for jack */
+#endif
 
 #ifdef CONFIG_PM
 	unsigned long power_on_acct;
